apps/navstar_overlay/test/navstar_overlay_netlink_SUITE.erl

```diff
@@ -103,6 +103,7 @@ test_ip_cmds(Config) ->
 
     %% Test5: ip route replace
     DstIP = {192,168,65,91},
+    {ok, _} = navstar_overlay_netlink:ipaddr_replace(Pid, IP, PrefixLen, ?IFNAME),
```

Comment:
We don't necessarily need this change because these tests are part of the same test case. However, it is harmless and probably would avoid issues such as splitting these tests into multiple test cases.